config/routes.rb

```diff
@@ -80,6 +80,10 @@
   get "books/:slug/extras",      to: "books#extras",          as: :book_extras
 
 
+  # Videos
+  get "watch/:slug", to: "videos#show", as: :video
```

Comment:
I think we should:

- [ ] add `/videos` to the routes
- [ ] move the contents of of `about#watch` to `videos#index`
- [ ] move the contents of of `about/index` to `videos/index`
- [ ] add a `redirect` route from `watch` to `videos` (like `listen` to `podcast`)